### PR TITLE
Add Pulsar Summit Europe Schedule blog

### PR DESCRIPTION
**Pull request by Sherlock113**
### Documentation

- [ ] `doc` 
- [ ] `doc-required` 
- [x] `doc-not-needed` 
- [ ] `doc-complete` 

Added the Pulsar Summit Europe Schedule blog and updated the banner. Previews:
Screenshot 2023-04-11 at 14 55 37
Screenshot 2023-04-11 at 14 55 44
Screenshot 2023-04-11 at 14 55 56
Screenshot 2023-04-11 at 14 55 27
Screenshot 2023-04-11 at 14 55 06

### CODE REVIEW

**Comment by Sherlock113** [comment]
@jak78 @tisonkun @KarinLanders PTAL. Thanks.

**Review by jak78 — APPROVED** [review]

**Review by Anonymitaet — APPROVED** [review]

File: blog/2023-04-11-pulsar-summit-europe-2023-schedule-announced.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+This year, Pulsar Virtual Summit Europe will include tech deep dives, use cases, best practices, and insights into Pulsar’s global adoption and thriving community. Take a sneak peek below at a few of the featured sessions.
+
+### Challenges of Hosting a Pulsar-as-a-Service Platform Under a Shared Responsibility Model
+
+Edgaras Petovradzius, Senior Engineer, the LEGO Group
+
+Mathias Ravn Tversted, Engineer, the LEGO Group 
+
+This talk will explore the challenges the LEGO Group encountered to host and manage Pulsar-as-a-Service across multiple domains, and how they collaborated with StreamNative in the process. It will highlight how the organization used OAuth2 for authentication and its self-service management platform for access control of Pulsar resources. The speakers will also discuss their observability tooling, incorporating cloud-native tools like Prometheus and OTel collectors, as well as internal tools for metrics and an ELK stack. Finally, the presentation will share best practices for using Pulsar clients in various programming languages and explore alternative methods of producing and consuming messages.
```
[the remark this answers is not in the document]
Comment (Sherlock113) [reply blog/2023-04-11-pulsar-summit-europe-2023-schedule-announced.md]: Changed.

**Comment by tisonkun** [comment]
Pending for merging...

**Comment by tisonkun** [comment]
@Sherlock113 Thanks for preparing this blog!

**Pull request merged**